src/styles/What.css

```diff
@@ -337,103 +297,124 @@
         width: 100px;
         height: 100px;
     }
-    .endingBlurbs{
+    .endingBlurbText{
         font-size: 13pt;
         width: 57.5vw;
     }
-    #wwdRightBlurb {
-      width: 75px;
-      height: 75px;
-      border-radius: 50%;
-      margin-left: 40px;
-      box-shadow: 5px 5px 5px grey;
-    }
     .Catch{
         font-size: 20pt;
     }
     #catchDiv{
         padding: 15px;
     }
     #title{
-      font-size: 22px;
+      font-size: 5vw;
+    }
+    #wwdLeftBlurb {
+      width: 65px;
+      height: 65px;
+    }
+    #wwdRightBlurb {
+      width: 65px;
+      height: 65px;
     }
 }
 
 @media (max-width: 1800px) and (min-width: 751px){
```

Comment:
What.css: Is `@media (max-width: 1800px) and (min-width: 751px)` meant to be `@media (max-width: 900px) and (min-width: 751px)`?